spec/lib/api/v3/projects/project_representer_spec.rb

```diff
@@ -65,6 +65,33 @@
         expect(subject).to have_json_path('_links/self/title')
       end
 
+      describe 'create work packages' do
+        let(:user) do
+          FactoryGirl.build(:user, member_in_project: project, member_through_role: role)
+        end
+        let(:project) { FactoryGirl.create(:project) }
+        let(:representer) { described_class.new(project, current_user: user) }
+        context 'user allowed to create work packages' do
+          let(:role) { FactoryGirl.create(:role, permissions: [:add_work_packages]) }
+
+          it { is_expected.to have_json_path('_links/createWorkPackage') }
+          it { is_expected.to have_json_path('_links/createWorkPackage/href') }
```

Comment:
You only need the second expectation.

Bonus points for checking that the link is actually the one that the path helper would return.

*Note: Taking bonus points is mandatory ;-)*